File: src/_2021/_1a.js
// December 1st, 2021
const fs = require("fs");

const list = fs.readFileSync("input/2021/1.txt", "utf8").split('\r\n').map(x => parseInt(x));
let count = 0;
for (var i = 1; i < list.length; i++) {
	if (list[i] > list[i - 1]) count++;
}
console.log(count);

File: src/_2021/_1a.py
# December 25th, 2021

file = open("input/2021/1.txt", "r")
lines = file.read().split("\n")

count = 0
for i in range(1, len(lines) - 1):
	if int(lines[i]) > int(lines[i - 1]):
		count += 1
print(count)
